solutions/cses-1715.mdx

```diff
@@ -13,7 +13,7 @@ We are essentially counting how many distinct strings we can make by permuting t
 
 For example, consider the string $aab$, and supposed we denote the $a$'s that sppear in the string in order of which they appear, so the original string becomes $a_1a_2b$. One possible permutation is $a_2a_1b$. When we remove these subscripts, these two strings become the same thing. 
 
-Therefore, we will need to divide those possibilities off. The total number of permutations is $n!$. The number of ways a letter can permute is also just the factorial of the number of times it appears in the string. 
+Therefore, we will need to divide off the permutations of each repeating letter for this. The total number of permutations is $n!$. The number of ways a letter can permute is also just the factorial of the number of times it appears in the string. 
```

Comment:
> the permutations of each repeating letter for this

what

Reply:
how to english 

:divide off the number of arrangements for each individual letter"?

Reply:
what does permutations of a letter mean?
if it's a single character, there's just 1 permutation

Reply:
each type of letter in the alphabet

Reply:
so say if there were 2 `a`s, we'd divide by $2!$?

Reply:
yeah

Reply:
better?